src/components/UNSTABLE_Pagination/Pagination.stories.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file Unstable pagination stories.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { UNSTABLE_Pagination, PageSelector } from '../..';
+
+import { components } from '../../../.storybook';
+
+storiesOf(components('UNSTABLE Pagination'), module)
+  .addDecorator(story => <div style={{ width: '800px' }}>{story()}</div>)
+  .add('default', () => (
+    <UNSTABLE_Pagination totalItems={350} pageSizes={[10, 15, 20, 25]}>
+      {({ currentPage, onSetPage, totalPages }) => (
```

Comment:
I'm not familiar with this pattern - It looks like a render prop for `children`, which makes sense, but I'm curious to hear your reasoning behind it.

Reply:
This allows the parent component to share information (current page, etc) with children, and then inversely allows info to be shared back (setting current page, etc). While it is a more hands-on approach for consumers, it's super flexible for them.